distribution/changelog.txt

```diff
@@ -23,6 +23,7 @@
 - Fix: [#22395, #22396] Misaligned tick marks in financial and guest count graphs (original bug).
 - Fix: [#22457] Potential crash opening the scenario select window.
 - Fix: [#22520] Virtual floor no longer appears when holding modifier keys during track construction.
+- Fix: [#22527] Forcing an element type to “wall” via the console can crash the game.
```

Comment:
Don't you mean scripting? The fix is in a scripting function

Reply:
Oops, I was testing it via the console which exposes the script engine (not the in game console). I'll update everything.